.travis.yml

```diff
@@ -26,7 +26,7 @@ before_script:
 - composer --version
 - node --version
 - npm --version
-script: npm run lint && npm run compile:pl
+script: npm run lint && npm run test:unit && npm run compile:pl
```

Comment:
Nice! So we're including unit tests by default whereas pa11y is optional?

Reply:
pa11y current "fails". I'll get it included in Travis builds once it's passing.